README.md

```diff
@@ -53,10 +53,16 @@ set up on github pages
 
 👤 **Ezekiel Utshudi**
 
+👤 **Sangwa Didier**
+
 - GitHub: [@EzekielUtshudi](https://github.com/EzekielUtshudi)
 - Twitter: [@UtshudiEzekiel](https://twitter.com/UtshudiEzekiel)
 - LinkedIn: [@UtshudiEzekiel](https://www.linkedin.com/in/ezekiel-utshudi-195782162/)
 
+- GitHub: [@SangwaDidier](https://github.com/sangwa7)
+- Twitter: [@DidierSangwa](https://twitter.com/Didiersangwa)
+- LinkedIn: [@DidierSangwa](https://www.linkedin.com/in/Sanwa-didier/)
+
```

Comment:
- [ ] Kindly move name  of author to move to the second author's  social media links
```suggestion
👤 **Sangwa Didier**
- GitHub: [@SangwaDidier](https://github.com/sangwa7)
- Twitter: [@DidierSangwa](https://twitter.com/Didiersangwa)
- LinkedIn: [@DidierSangwa](https://www.linkedin.com/in/Sanwa-didier/)

```